src/descriptors/power_system_structs.json

```diff
@@ -1319,6 +1319,17 @@
                 "data_type": "TwoPartCost",
                 "comment": "Operation Cost of Generation [`TwoPartCost`](@ref)"
             },
+            {
```

Comment:
Should this value default to 100.0 so that users are not required to provide it?  (for all devices)

Reply:
There shouldn't be a default for this. It only makes sense for very little systems, but the data in all formats has this column

Reply:
It is possible. I didn't include it as a default since the parameter is available in the datasets. However, if you check those columns in .m files or in .raw files, you most likely see everything at 100.0.

It's up to the discussion. The alternative is default to 100.0, and in the parsing, if it's available, do the update. Considering the parsing available, this will be updated all the time. The key difference will be when constructing it directly via the struct itself.

Reply:
One of my thoughts is that we are continually breaking our users.  Maybe this is OK until we tag 1.0.

Reply:
@daniel-thom agreed, this is why we have time until September to tag 1.0 hopefully having developed most the code for operations + dynamics will allow covering most of the requirements.